config/musonza_chat.php

```diff
@@ -2,6 +2,7 @@
 
 return [
     'user_model' => 'App\User',
+    'db_connection' => 'mysql',
```

Comment:
Can you leave this null so that if not set we read from `config('database.default')`